src/PressButtonLogic.js

```diff
@@ -0,0 +1,160 @@
+const canConcat = (value, state) => {
```

Comment:
By default, all the files should start with a lower case letter. The only exception (for now at least) are the files with components. Please rename all the files accordingly.